cmd/sysllsp/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/anz-bank/sysl/internal/jsonrpc2"
+	"github.com/anz-bank/sysl/pkg/lspimpl/lsprpc"
+)
+
+func main() {
+	// start the lsp debug server here
+	ctx := context.Background()
+
+	/*
+		di := debug.GetInstance(ctx)
+		if di != nil {
+			closeLog, err := di.SetLogFile(s.Logfile)
+			if err != nil {
+				return err
+			}
+			defer closeLog()
+			di.ServerAddress = s.Address
+			di.DebugAddress = s.Debug
+			di.Serve(ctx)
+			di.MonitorMemory(ctx)
+		}
+	*/
```

Comment:
I discussed with @nydrani, so the forked code are the LSP framework that gopls use and the commented code are golang specific (processing data modelling of golang language construct), so in the future it can be reused for sysl instead. I'll put `TODO`s on the commented code.